pkg/release/version.go

```diff
@@ -0,0 +1,15 @@
+package release
+
+type Version string
+
+func NewStableVersion() Version {
+	return Version(":stable")
```

Comment:
This won't be correct here, we do not need the `:` anymore when using helmclient.